content/en/docs/Writing policies/verify-images.md

```diff
@@ -4,4 +4,145 @@ description: Check image signatures and add digests
 weight: 3
 ---
 
-TBD...
\ No newline at end of file
+{{% alert title="Note" color="warning" %}}
+Image verification is an **preview** feature. It is not ready for production usage and will likely change. 
```

Comment:
"...is a preview feature."

Reply:
Or is it better to call this an alpha feature?

Reply:
I want back and forth....`alpha` is fine.